src/state/helpers.ts

```diff
@@ -5,11 +5,8 @@ import { assertUnreachable } from '../lib/utils'
 export const isCrossAccountTradeSupported = (swapperName: SwapperName) => {
   switch (swapperName) {
     case SwapperName.Thorchain:
-      return true
-    // NOTE: Before enabling cross-account for LIFI and OneInch - we must pass the sending address
-    // to the swappers up so allowance checks work. They're currently using the receive address
-    // assuming it's the same address as the sending address.
```

Comment:
Removing this comment now, as LiFi is fixed in this PR (the receive address was actually already set correctly), and 1inch in https://github.com/shapeshift/web/pull/5834.